boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -67,4 +114,27 @@ config DOMAIN_CPUAPP_BOARD
 	  image application where one or more images should be located on
 	  another board.
 
+if BT
+
+# Set defaults to allow fast MCUmgr transfers for FOTA over Bluetooth
+config BT_BUF_ACL_TX_SIZE
+	default 502
+
+config BT_BUF_ACL_RX_SIZE
+	default 502
+
+config BT_BUF_EVT_RX_SIZE
+	default 255
+
+config BT_BUF_EVT_DISCARDABLE_SIZE
+	default 255
+
+config BT_BUF_CMD_TX_SIZE
+	default 255
+
+config BT_CTLR_DATA_LENGTH_MAX
```

Comment:
It seems that the HCI RPMsg child image does not build for Thingy:53 because of RAM overflows.

```region `RAM' overflowed by 12120 bytes```

Unfortunately, the HCI RPMsg sample in the upstream sets the number of buffers to 16 and this configuration cannot be changed using defaults in this file. We may consider adding the hci_rpmsg.conf to each Bluetooth sample to make it work.

Reply:
Fixed to allow 3 connections by default, builds with:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      184820 B       222 KB     81.30%
             RAM:       47016 B        64 KB     71.74%
           SRAM1:          0 GB        64 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
```
Could possibly extend this to 4 connections?

Reply:
Thanks, that's a better solution. We will see during our tests if 3 connections are enough. Being too close to 100% of RAM usage may trigger more maintenance work in the trigger as the Zephyr Bluetooth stack keeps growing and could potentially trigger RAM overflow after some time.